mat64/dense.go

```diff
@@ -65,6 +65,13 @@ type Dense struct {
 	capRows, capCols int
 }
 
+// NewDense creates a new matrix of type *Dense with dimensions r and c. If the
+// mat argument is nil, a new data slice is allocated.
+//
+// The data must be arranged in row-major order, i.e. the (i*c + j)-th element
+// in mat is the {i, j}-th element in the matrix.
+//
+// See Reset for how to use zero-sized matrix.
```

Comment:
I think this last line should go away. If you don't want to prepare an example test, please file an issue for that and we can do that later.